crates/katana-core/src/sequencer.rs

```diff
@@ -120,6 +122,13 @@ impl KatanaSequencer {
             .unwrap()
             .get_storage_at(contract_address, storage_key)
     }
+
+    pub async fn starknet_get_contract_class(
+        &self,
+        class_hash: &ClassHash,
```

Comment:
```suggestion
        class_hash: ClassHash,
```

we can pass by value here